### PR TITLE
[Feature] Add importing amass animation

### PR DESCRIPTION
**Pull request by HaiyiMei**
- supporting import animation from amass file and apply it to the actor
- add a sample code `07_amass.py`

### CODE REVIEW

File: samples/blender/07_amass.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+"""
+>>> python -m samples.blender.07_amass
+
+This is a script to demonstrate importing Amass motion and applying it to SMPL-XL model.
+Before running this script, please download SMPL-XL model and Amass dataset first,
```
Comment (Maoxie) [inline samples/blender/07_amass.py:5]: For users not knowing SMPL-XL, it's better to offer a ref link to the paper here.

File: src/XRFeitoriaBpy/core/factory.py [hunk 2]
```diff
@@ -998,28 +998,17 @@ def import_glb(glb_file: str) -> None:
...
-        actor_name: str,
-    ) -> None:
+    def import_mo_json(mo_json_file: Path, actor_name: str) -> None:
         """Import an animation from json, and apply the animation to the given actor.
 
         Args:
             mo_json_file (Path): json file path.
             actor_name (str): Name of the actor.
         """
         motion_data = json.load(Path(mo_json_file).open('r'))
```
Comment (Maoxie) [inline src/XRFeitoriaBpy/core/factory.py:1008]: Opening a file without close.

File: samples/blender/07_amass.py (hunk 1 above)
Comment (HaiyiMei) [reply samples/blender/07_amass.py]: Fair suggestion. Add paper links in [`b54c643` (#12)](https://github.com/openxrlab/xrfeitoria/pull/12/commits/b54c643570638bb0ed3d5b31cdbfaad1ba086b45#diff-9be48bfb056bce8b28b26d9d062d75efd32a4312065b77ec113dda52d7700753).

File: src/XRFeitoriaBpy/core/factory.py (hunk 2 above)
Comment (HaiyiMei) [reply src/XRFeitoriaBpy/core/factory.py]: Fixed in [`b54c643` (#12)](https://github.com/openxrlab/xrfeitoria/pull/12/commits/b54c643570638bb0ed3d5b31cdbfaad1ba086b45#diff-0cae5b014ff6e456cc55ea817c51670678870333f7a3cb0b17a46fa974562ebf)

**Review by Maoxie — APPROVED** [review]

**Pull request merged**